model-client/src/commonMain/kotlin/org/modelix/model/client2/ModelClientV2.kt

```diff
@@ -80,6 +80,9 @@ class ModelClientV2(
     }
 
     suspend fun updateUserId() {
+        if (userId !== null) {
```

Comment:
This changes the behavior of `updateUserId()` if no `userId` was provided in the constructor. The constructor parameter should be a separate variable so that it's possible to distinguish whether the initial value was provided in the constructor or by the server.

Reply:
You are right. I missed, that `updateUserId` is a public function.

Reply:
Done. It can be reviewed again.

I also added the possibility to change the user ID afterward.